Doppler.HtmlEditorApi/Repositories.DopplerDb/DopplerCampaignContentRepository.cs

```diff
@@ -104,23 +104,6 @@ public async Task CreateCampaignContent(string accountName, ContentData content)
         };
 
         await _dbContext.ExecuteAsync(insertContentQuery);
-
-        if (content is BaseHtmlContentData baseHtmlContentData)
-        {
-            var updatePreviewImageQuery = new UpdateCampaignPreviewImageDbQuery(
-                baseHtmlContentData.CampaignId,
-                baseHtmlContentData.PreviewImage);
-            await _dbContext.ExecuteAsync(updatePreviewImageQuery);
-        }
-
-        var updateCampaignStatusQuery = new UpdateCampaignStatusDbQuery(
-            SetCurrentStep: 2,
-            SetHtmlSourceType: UpdateCampaignStatusDbQuery.TemplateHtmlSourceType,
```

Comment:
You are not using the constant `UpdateCampaignStatusDbQuery.TemplateHtmlSourceType` anymore, should we deleted it? maybe move it to another place?

Reply:
Maybe I can move it to the controller and use ir in the method call that we're [discussing here](https://github.com/FromDoppler/doppler-html-editor-api/pull/111#discussion_r870661132).

Reply:
🚀